chia/util/initial-config.yaml

```
@@ -491,7 +491,10 @@ wallet:
   full_node_peer:
     host: *self_hostname
     port: 8444
-
+  # The path of NFT off-chain metadata cache
+  nft_metadata_cache_path: "./nft_cache"
```

See the other comment about `chia.util.path.path_from_root()`.

```suggestion
  nft_metadata_cache_path: "nft_cache"
```

---

./ is used to distinguish the relative path and absolute path. User may not want to cache the off-chain metadata on their system disk.

---

This is no different than our other big stuff.  It need not be defaulted differently here.  Using `./` just means it will go any number of places depending what directory the user is in when they run chia.